package NYP.NServer.NObjects.NProto;

////////////////////////////////////////////////////////////////////////////////

message TSelectObjectsContinuationToken
{
    // Indicates version of token field values.
    optional int32 version = 1;

    // The first object returned will have db key strictly greater than
    // the key produced by a combination of the following fields.
    optional string parent_id = 3;
    optional string object_id = 2;
}

message TSelectObjectHistoryContinuationToken
{
    // Number of earlier returned history events.
    optional int32 offset = 1;
}

message TWatchQueryContinuationToken
{
    // Offset for each tablet.
    repeated int64 row_offsets = 1;
}

////////////////////////////////////////////////////////////////////////////////
